### PR TITLE
css-canvas-medical - Yuliia P.

### PR DESCRIPTION
**Pull request by yuliia-p**
Screen Shot 2022-07-04 at 9 46 11 AM

### CODE REVIEW

**Comment by yuliia-p** [comment]
Screen Shot 2022-07-07 at 11 08 28 PM

File: css-canvas-medical/styles.css [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+  font-size: 0.9rem;
+}
+
+.logo {
+  max-width: 40%;
+  margin-bottom: 30px;
+  margin-top: 15px;
+}
+
+.column-two-fourths {
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:40]: `column-half`?

File: css-canvas-medical/styles.css [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+}
+
+div.column-two-fourths > p {
+  font-size: 0.8rem;
+}
+
+.column-one-fourth {
+  flex-basis: calc(100% / 4);
+  line-height: 20px;
+  margin-left: 50px;
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:57]: Don't pollute your layout classes, make these into a separate rule

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-canvas-medical/styles.css (hunk 1 above)
Comment (codingcodymiller) [reply css-canvas-medical/styles.css:40]: This still needs to be addressed.

Comment (codingcodymiller) [reply css-canvas-medical/styles.css:40]: This still needs to be addressed.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by yuliia-p** [comment]
Screen Shot 2022-08-09 at 11 37 34 AM
Screen Shot 2022-08-09 at 11 37 42 AM

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**